Microservices/event-service/src/main/java/app/sportahub/eventservice/model/event/Location.java

```diff
@@ -0,0 +1,40 @@
+package app.sportahub.eventservice.model.event;
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder(setterPrefix = "with")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class Location {
+
+    @NotBlank(message = "Location name must be provided")
+    private String name;
+
+    @NotBlank(message = "Street number must be provided")
```

Comment:
Can you make it nullable as well? Event can be taken place at a Park where no street is given.

Reply:
Good point, done